modules/Core/pages/panel/minecraft_servers.php

```diff
@@ -600,6 +611,9 @@
     $status_page = DB::getInstance()->get('settings', ['name', 'status_page'])->results();
     $status_page = $status_page[0]->value;
 
```

Comment:
Can this be done as a one liner or has it be done like this for readability?

Reply:
Just for consistency